.github/stale.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+daysUntilClose: 14
+
+# Issues or Pull Requests with these labels will never be considered stale
+exemptLabels:
+  - "enhancement"
+
+# Label to use when marking as stale
+staleLabel: "awaiting input"
```

Comment:
What about just "stale"? I can't really think of anything better, although it seems like something's just on the tip of my tongue. "awaiting input" seems... off.